sycl/test-e2e/KernelFusion/sync_two_queues_event_dep.cpp

```diff
@@ -1,8 +1,6 @@
+// REQUIRES: fusion
 // RUN: %{build} -o %t.out
 // RUN: env SYCL_RT_WARNING_LEVEL=1 %{run} %t.out 2>&1 | FileCheck %s
-// UNSUPPORTED: hip
-// For this test, complete_fusion must be supported.
```

Comment:
Nit: Can we preserve the comment here and in the next test?

Reply:
Done